pageserver/src/layered_repository/ephemeral_file.rs

```diff
@@ -0,0 +1,298 @@
+//! Implementation of append-only file data structure
```

Comment:
Why ephemeral file is called append-only?
As far as I understand it is used for page cache and each block can be overwritten multiple times